packages/protocol/contracts/governance/Validators.sol

```diff
@@ -35,7 +37,7 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
     string identifier;
     string name;
     string url;
-    bytes publicKey;
+    bytes publicKeysData;
```

Comment:
Another thought on this. Probably out of scope of this PR since we're not actually checking POPs, but do you agree that once we do we wouldn't need to store the POP itself, since we've already verified it at registration time?

Reply:
Totally agree.